th2_data_services/interfaces/events_tree/events_tree_collection.py

```diff
@@ -137,7 +163,13 @@ def _build_trees(self, nodes: Dict[Optional[str], List[Node]]) -> None:
         nodes.pop(None)
 
         self._roots = [EventsTree(root) for root in roots]
+
         self._detached_nodes = nodes
+        if self._detached_nodes:
+            w = "You have created a ETC with detached events."
```

Comment:
Let's use more scary expression.
It's a warning, not an info message.

We should say user that something went wrong!

Reply:
Will fixed in next commit.

Reply:
Warnings message was changed to "Tree created with detached events because some events have no parentsId. Check your data."
https://github.com/th2-net/th2-data-services/commit/0460ed688d47f24a298ed465e68c81b9cfbf7118